test/performance/odp_packet_gen.c

```diff
@@ -114,6 +121,10 @@ static void print_usage(void)
 	       "                          addresses (no spaces), one address per packet IO\n"
 	       "                          interface e.g. AA:BB:CC:DD:EE:FF,11:22:33:44:55:66\n"
 	       "                          Default per interface: 02:00:00:A0:B0:CX, where X = 0,1,...\n"
+	       "  -v, --vlan <tpid:tci>   VLAN configuration. Comma-separated list of VLAN TPID:TCI\n"
+	       "                          values in hexadecimal. For example:\n"
+	       "                          VLAN 200 (decimal):          8100:c8\n"
+	       "                          Double tagged VLANs 1 and 2: 88a8:1,8100:2\n"
```

Comment:
You could mention the order of the VLAN headers here.

Reply:
fixed in v2